packages/p2p/src/stores/buy-sell-store.js

```diff
@@ -406,6 +431,32 @@ export default class BuySellStore extends BaseStore {
         this.items = items;
     }
 
+    @action.bound
+    setLocalCurrencies(local_currencies) {
+        const currency_list = [];
+
+        local_currencies.forEach(currency => {
+            const { display_name, has_adverts, is_default, symbol } = currency;
+
+            if (is_default) this.setSelectedLocalCurrency(currency.symbol);
```

Comment:
```suggestion
            if (is_default) this.setSelectedLocalCurrency(symbol);
```
shouldn't we just pass symbol since it already has been destructured